File: TuristickaAgencija.Model/Vodic.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace TuristickaAgencija.Model
{
    public partial class Vodic
    {
        public int Id { get; set; }
        public string Ime { get; set; }
        public string Prezime { get; set; }
        public string Kontakt { get; set; }
        public string Jmbg { get; set; }
        public byte[] Slika { get; set; }

        //public virtual ICollection<VodiciPutovanja> VodiciPutovanja { get; set; }
    }
}

File: UI/turisticka_agencija_admin/lib/models/Vodic.dart
import 'package:json_annotation/json_annotation.dart';

part 'Vodic.g.dart';

@JsonSerializable()
class Vodic {
    int id;
    String ime;
    String jmbg;
    String kontakt;
    String prezime;
    String slika;

    Vodic({required this.id,required this.ime,required this.jmbg,required this.kontakt,
        required this.prezime,required this.slika});

    factory Vodic.fromJson(Map<String, dynamic> json) => _$VodicFromJson(json);

    Map<String, dynamic> toJson() => _$VodicToJson(this);
}